app/serializers/api/v2/event_serializer.rb

```diff
@@ -0,0 +1,18 @@
+module Api
+  module V2
+    class EventSerializer < ApplicationSerializer
+      attributes :id, :sequence_id, :kind, :map_id, :created_at
+
+      has_one :actor, serializer: UserSerializer, root: 'users'
+      has_one :map, serializer: MapSerializer
+
+      def actor
+        object.user || object.eventable.try(:user)
+      end
+
+      def map
+        object.eventable.try(:map) || object.eventable.map
+      end
+    end
+  end
+end
```

Comment:
Definitely. 

In node, you have versioning like x.y.z. If you remove/change an existing API, you bump the major version (x). If you add a new API, you bump the minor version (y). If you just change internals/bugfixes, you only need to bump the patch version (z).

I don't propose we do minor/patch versions, but I do think we should follow the no removals/changes to existing APIs once this is released.